modules/nf-core/doubletdetection/main.nf

```diff
@@ -1,33 +1,30 @@
 process DOUBLETDETECTION {
-    tag "$meta.id"
+    tag "${meta.id}"
     label 'process_medium'
 
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/87/878a0582c1de0ad7370ad1fbdebd7e786c77d29b064e10a7c09c35a9df3bfb97/data' :
-        'community.wave.seqera.io/library/anndata_louvain_numpy_pip_pruned:9ff7bfd3c5201947' }"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'oras://community.wave.seqera.io/library/pyyaml_pip_doubletdetection:d67fd9203a935dfe'
```

Comment:
Please use https. See here on how to get the right URL https://nf-co.re/docs/tutorials/nf-core_components/using_seqera_containers